Smarty/templates/modules/Calendar4You/CalendarView.tpl

```diff
@@ -456,6 +456,23 @@ function hideITSEventInfo(){
     jQuery('#event_info_content').html('');
 
 }
+
```

Comment:
Let's keep investigating and learning. Most of the calendar functionality is new to me too.